README.md

```diff
@@ -3,64 +3,8 @@
 The espejo tool (which means 'mirror' in Spanish) syncs objects from a SyncConfig CRD to multiple namespaces. The idea is to replace OpenShift's [project templates](https://docs.openshift.com/container-platform/3.9/admin_guide/managing_projects.html#modifying-the-template-for-new-projects) with a more flexible and robust solution.
 
 ## CustomResourceDefinitions
-The operator introduces a CRD called `SyncConfig` to configure the objects which should be synced:
-```yaml
-apiVersion: sync.appuio.ch/v1alpha1
-kind: SyncConfig
-metadata:
-  name: example
-spec:
-  forceRecreate: false
-  namespaceSelector:
-    labelSelector:
-      matchExpressions:
-      - key: customer
-        operator: Exists
-      matchLabels:
-        some-label: value
-    matchNames:
-    - myproject
-  items:
-  - apiVersion: v1
-    kind: Service
-    metadata:
-      name: glusterfs-cluster
-    spec:
-      type: ClusterIP
-      clusterIP: None
-      ports:
-      - port: 49152
-        targetPort: 49152
-        protocol: TCP
-  - apiVersion: v1
-    kind: Endpoints
-    metadata:
-      name: glusterfs-cluster
-    subsets:
-    - addresses:
-      - ip: 172.28.54.121
-      - ip: 172.28.54.122
-      - ip: 172.28.54.123
-      ports:
-      - port: 49152
-        protocol: TCP
-  - apiVersion: extensions/v1beta1
-    kind: NetworkPolicy
-    metadata:
-      name: allow-from-same-namespace
-    spec:
-      ingress:
-      - from:
-        - podSelector: {}
-      podSelector: {}
-      policyTypes:
-      - Ingress
-  deleteItems:
-  - apiVersion: v1
-    kind: ConfigMap
-    name: some-name
-```
-This `SyncConfig` will create a `Service`, `Endpoints` and `NetworkPolicy` object in all namepsaces which mach the [label selector](https://v1-9.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.9/#labelselector-v1-meta) OR one of the name selectors.
+The operator introduces a CRD called `SyncConfig` to configure the objects which should be synced (see [example](deploy/crds/sync_v1alpha1_syncconfig_cr.yaml)).
+[This `SyncConfig`](deploy/crds/sync_v1alpha1_syncconfig_cr.yaml) will create a `Service`, `Endpoints` and `NetworkPolicy` object in all namepsaces which mach the [label selector](https://v1-9.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.9/#labelselector-v1-meta) OR one of the name selectors.
```

Comment:
The domain `v1-9.docs.kubernetes.io` yields an invalid certificate warning when opened in a browser. It looks like the v1.9 is no longer available.

Reply:
Good catch (I also updated the OpenShift links).